visidata/loaders/postgres.py

```diff
@@ -18,13 +18,12 @@ def codeToType(type_code, colname):
 def openurl_postgres(url, filetype=None):
     import psycopg2
 
-    dbname = url.path[1:]
     conn = psycopg2.connect(
-                user=url.username,
-                dbname=dbname,
-                host=url.hostname,
-                port=url.port,
-                password=url.password)
+                user=url.given.split('/')[2].split(':')[0],
```

Comment:
Probably we can do something like `url = urlparse(url.given)`.  Maybe even this should be done automatically for the `openurl_` calls.

Reply:
We do! This is in `vdPath().scheme`, but the result was very weird, lemme paste it. One moment.

Reply:
```
ParseResult(scheme='postgres', netloc='kefala:fakeplastictrees@localhost:56714', path='/test', params='', query='', fragment='')
```

Ah, the issue is that `vdPath().scheme` only contains that `scheme` attribute.

Reply:
Okay, this is slightly better to work with. I can work with this. I shall return tomorrow with a better update.